src/hooks/progress/use-tutorial-progress.ts

```diff
@@ -14,9 +14,11 @@ import {
 import { PROGRESS_BATCH_QUERY_ID, TUTORIAL_PROGRESS_SINGLE_QUERY_ID } from './'
 import { useHasWaitedForQuery } from 'hooks/use-has-waited-for-query'
 
+type QueryResult = TutorialProgressStatus | null
```

Comment:
Realized this type was incorrect, as the queried result can be `null` if there is not progress record. I think I had this at some previous point, but reverted it at some point during the batch work.